src/pybind/mgr/dashboard/CMakeLists.txt

```diff
@@ -104,3 +95,16 @@ add_custom_target(mgr-dashboard-frontend-build
   WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/src/pybind/mgr/dashboard/frontend)
 add_dependencies(tests mgr-dashboard-frontend-build)
 endif(WITH_MGR_DASHBOARD_FRONTEND AND NOT CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|AARCH64|arm|ARM")
+
+if(WITH_TESTS)
+  include(AddCephTest)
+  if(WITH_PYTHON2)
+    list(APPEND dashboard_tox_envs py27-cov py27-lint py27-check)
```

Comment:
We may get rid of this as soon as I rebase my PR https://github.com/ceph/ceph/pull/28696 with your changes.

Reply:
yup, that's the plan.